uint8 motion_command # 0-> stop_moving_for, 1-> go forward, 2-> go backward, 3-> spin left, 4-> spin right
float32 robot_speed
